src/components/Navbar/index.tsx

```diff
@@ -21,27 +35,38 @@ const Navbar: React.FC = () => {
 
                 <ul className={'lg:flex space-x-4'}>
                     <li className="relative group">
-                        <Button type="button" onClick={toggleDropdown} className="text-white focus:outline-none">
-                            <div className="flex items-center space-x-2">
-                                <ProfileIcon firstName={firstName} lastName={lastName} size={50} />
+                        {isLoggedIn ? (
+                            <Button type="button" onClick={toggleDropdown} className="text-white focus:outline-none">
+                                <div className="flex items-center space-x-2">
+                                    <ProfileIcon firstName={firstName} lastName={lastName} size={50} />
+                                    <span> {firstName}</span>
+                                    <svg
+                                        xmlns="http://www.w3.org/2000/svg"
+                                        className="h-5 w-5 inline-block ml-1 transform group-hover:rotate-180 transition-transform"
+                                        fill="none"
+                                        viewBox="0 0 24 24"
+                                        stroke="currentColor"
+                                    >
+                                        <path
+                                            strokeLinecap="round"
+                                            strokeLinejoin="round"
+                                            strokeWidth={2}
+                                            d="M19 9l-7 7-7-7"
+                                        />
+                                    </svg>
```

Comment:
Move this to svgs

Reply:
not meeded

Reply:
Why?
this will increase the lines of code in this component and component size should be as small as possible